web-app/preprocess/classification.py

```diff
@@ -0,0 +1,74 @@
+import sys
+import nltk
+import string
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import os
+import pickle
+import numpy as np
+import fasttext.util
+from scipy.spatial.distance import cosine
+# Remove "other" from list of stopwords in order to classify crime types such as "other crime", "etc"
```

Comment:
nit: is this a todo? Maybe mark as such?

Reply:
Oh this is more like letting the user of the code know this code properly works once other is deleted from list of stop words downloaded by nltk!